scripts/pypi_helper.py

```diff
@@ -1,26 +1,26 @@
-import requests as request
+import requests
 
-import ldm.invoke._version as version
+from ldm.invoke.__init__ import __app_name__, __version__
```

Comment:
This is a simpler version:
```
from ldm.invoke import __app_name__, __version__
```

Reply:
kk, I was scared this would already import more 🙈